app/views/configure/reading.phtml

```diff
@@ -136,6 +136,15 @@
 					</select>
 				</div>
 			</div>
+			<div class="form-group">
+				<label class="group-name" for="show_article_icons"><?= _t('conf.reading.article.icons') ?></label>
+				<div class="group-controls">
+					<select name="show_article_icons" id="show_article_icons" data-leave-validation="<?= FreshRSS_Context::userConf()->show_article_icons ?>">
+						<option value="t" <?= FreshRSS_Context::userConf()->show_article_icons === 't' ? ' selected="selected"' : '' ?> data-input-visible="true"><?= _t('conf.reading.article.icons.above_title') ?></option>	
```

Comment:
Trailing space here - please remember to use https://editorconfig.org in your editor / IDE.
Ref https://github.com/FreshRSS/FreshRSS/pull/6648/files#diff-3b728a20c27f783ad0ba0b0de17c4ca3d681985394672e9a0a5fd2be4d40e567

Reply:
should it not caught by the pipeline?

I use VS Code that respects the `.editorconfig`. 

Something is here broken I guess

Reply:
It's listed right at the start:

https://github.com/FreshRSS/FreshRSS/blob/0eeac4a669e46fb6521ae1765b23656c9afc43de/.editorconfig#L5-L7

Reply:
In VS Code, make sure you have the corresponding extension https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
We do not have a CI test for that at the moment (I plan to make an update for PHPCS soon, which might catch it)

Reply:
As promised https://github.com/FreshRSS/FreshRSS/pull/6666
This will now catch such cases as part of the CI, also also offer automatic fixes with
```sh
make test-all
# or isolated:
composer run-script phpcbf
```